frontend/online-shop/src/components/Products.tsx

```diff
@@ -0,0 +1,60 @@
+import { IconButton, ThemeProvider } from "@material-ui/core";
+import { useStyles } from "../styles/tableStyles";
+import InfoIcon from "@material-ui/icons/Info";
+import { Link } from "react-router-dom";
+import axios from "axios";
+import ProductService from "../services/ProductService";
+import React from "react";
+
+export interface ProductEntityProps {
+  id: number;
+  name: string;
+  description: string;
+  category: string;
+  price: number;
+  weight: number;
+  imageUrl: string;
+}
+
+export const TableProducts = (props: ProductEntityProps) => {
+  var productService = new ProductService();
```

Comment:
Try to avoid using var, use const instead

Reply:
Here's why: https://janakachathuranga.medium.com/why-we-use-const-and-avoid-var-3528ee550fb4

Reply:
Understood , refactored